### DISCUSSION TITLE
Skin Favoriting

### DISCUSSION THREAD

**Discussion by BlueFinnTuna** [ppy/osu#24267] [Ideas] [answered]
### **Favorite Skins**

This is definitely not a priority feature but I think it'd be nice to be able to favorite specific skins, maybe right-clicking them?
I've attached a quick mockup of what favorited skins could look like.

![mockup](https://github.com/ppy/osu/assets/71734783/eab7e0a7-a47c-478b-9f06-1d343d1377bf)

I don't know if it's worth the time or not but just an idea I had!

**Comment by peppy** [comment, marked as answer]
If you search you'll find similar requests and proposals. We do have plans to improve the system!
